docs/community.mdx

```diff
@@ -12,7 +12,7 @@ sidebar_position: 5
 
 ## Developing and contributing to ORAS
 
-If you are interested in developing or contributing to ORAS, you can check out the [CLI developer guide](https://oras.land/cli/5_developer_guide/).
+If you are interested in developing or contributing to ORAS, you can check out the [CLI developer guide](./cli/developer_guide/).
```

Comment:
```suggestion
If you are interested in developing or contributing to ORAS, you can check out the [CLI developer guide](./CLI/developer_guide/).
```
Although a minor nitpick, but matching the folder-name with links would make the docs look more consistent, and hopefully not give any errors in the future.

Reply:
Yeah, sure